main/parsers.h

```diff
@@ -51,6 +51,7 @@
 	PythonParser, \
 	RexxParser, \
 	RubyParser, \
+	RestParser, \
```

Comment:
Rest has a different meaning to a lot of people. Thought about calling it RstParser?